Wrappers/Python/test/test_DataProcessor.py

```diff
@@ -784,21 +781,21 @@ def setUp(self):
 
     @unittest.skipUnless(has_tomophantom and has_astra and has_nvidia, "Tomophantom or ASTRA GPU not installed")
     def test_CofR_image_sharpness_astra(self):
-        corr = CofR_image_sharpness(FBP=AstraFBP)
+        corr = CentreOfRotationCorrector.image_sharpness(backend='astra',FBP=FBP)
```

Comment:
Why does this have both backend and FBP set?

Reply:
Also I don't think FBP exists in this file?

Reply:
fixed!